Makefile

```diff
@@ -106,9 +106,14 @@ manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and Cust
 		webhook output:webhook:artifacts:config=config/components/webhook\
 		paths="./..."
 
+.PHONY: update-helm-crd
+update-helm-crd:
+	./hack/update-helm-crd.sh
+
 .PHONY: generate
 generate: controller-gen ## Generate code containing DeepCopy, DeepCopyInto, and DeepCopyObject method implementations.
 	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
+	$(MAKE) update-helm-crd
```

Comment:
I don't think we should do it in generate.

Reply:
Thanks. Fixed it.